scripts/openshift-tools-scripts.spec

```diff
@@ -46,6 +46,7 @@ cp -p monitoring/cron-send-docker-timer.py %{buildroot}/usr/bin/cron-send-docker
 cp -p monitoring/cron-send-docker-containers-usage.py %{buildroot}/usr/bin/cron-send-docker-containers-usage
 cp -p monitoring/cron-send-docker-dns-resolution.py %{buildroot}/usr/bin/cron-send-docker-dns-resolution
 cp -p monitoring/cron-send-docker-existing-dns-resolution.py %{buildroot}/usr/bin/cron-send-docker-existing-dns-resolution
+cp -p monitoring/cron-send-docker-oc-versions.py %{buildroot}/usr/bin/cron-send-docker-oc-versions
```

Comment:
This copies the file into the rpm build area, but there isn't an rpm package (or subpackage) where it's being claimed as a file.

Look at cron-send-docker-existing-dns-resolution.py for an example of how to have the RPM spec file claim the file to be laid down:

https://github.com/zgalor/openshift-tools/blob/ed63ee62c9427bf51b94003542719dfcd4d9a457/scripts/openshift-tools-scripts.spec#L193

Basically it has to be added to the `%files` section of a sub package (`monitoring-docker`).

Also, before asking for review, please always test creating a test RPM using tito.

```bash
# cd openshift-tools/script
# time tito build --test --rpm --output "$HOME/.tito-build"
```

This is also documented here (up to and including step 4):
https://github.com/openshift/openshift-tools/blob/stg/docs/build.adoc#build-openshift-tools-scripts--rpms